app/subscriber/src/components/sidebar/advanced-search/constants/SubMediaGroups.ts

```diff
@@ -12,6 +12,12 @@ export interface ISubMediaGroupItem {
  */
 export const SubMediaGroups = (
```

Comment:
This looks like maybe it should be a dictionary?  Not required, but wondering why an array?